src/index.css

```diff
@@ -1,52 +1,21 @@
-.nav-item {
-  background-color: #535bf2;
-}
-
+@import url('https://fonts.googleapis.com/css?family=Roboto');
 
-li{
-  list-style-type: none;
-  display: inline-block;
-  padding-right: 40px;
-  text-decoration: none;
+body:hover{
+  background-color: aliceblue;
 }
 
-li h1 {
-  padding-right: 1000px;
+/* form */
+.search{
+  background-color: none;
+  padding: 10px 10px;
+  margin-top: 30px;
 }
 
-@media (max-width: 600px)
-/* h1 {
-  
-} */
 
-/* button {
-  border-radius: 8px;
-  border: 1px solid transparent;
-  padding: 0.6em 1.2em;
-  font-size: 1em;
-  font-weight: 500;
-  font-family: inherit;
-  background-color: #1a1a1a;
-  cursor: pointer;
-  transition: border-color 0.25s;
-}
-button:hover {
-  border-color: #646cff;
-}
-button:focus,
-button:focus-visible {
-  outline: 4px auto -webkit-focus-ring-color;
-}
 
-@media (prefers-color-scheme: light) {
-  :root {
-    color: #213547;
-    background-color: #ffffff;
-  }
-  a:hover {
-    color: #747bff;
-  }
-  button {
-    background-color: #f9f9f9;
-  }
-} */
+@media screen and (max-width: 700px) {
+   .search{
+    padding:20px 150px;
+   }
```

Comment:
Format this properly. And also, there are so many many unnecessary spacing in this file. Trim that too